data/Danypooh.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Daniel Chapell",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Fullstack developer (loading)",
+  "avatar": "https://github.com/Danypooh.png"
```

Comment:
Thanks for your first contribution. 🎉  Currently, you can delete this code. You don't need to add `avatar` code. 
``` suggestion
 
```